tests/test_tasks.py

```diff
@@ -105,7 +105,7 @@ def test_sweep(fix_task_env):
     assert len(results) == 630 and results[3] == 0.0
 
 
-def test_docker_sweep(fix_task_env):
+def old_test_docker_sweep(fix_task_env):
```

Comment:
The rename effectively disables the test, doesn't it? If so, is there a reason to keep it around rather than just delete it?

Reply:
Ok I deleted it